custom/abdm/milestone_one/utils/abha_creation_util.py

```diff
@@ -0,0 +1,44 @@
+import logging
```

Comment:
All milestones have different usecases and if there is need to add or remove certain usecases, based on certifications received, it will be easier to locate the code milestone wise.

Reply:
Can we take that as supplementary change for another PR as I believe that should be more formal and researched. We will require to pick content from mobile app's doc.

Reply:
Cool.